src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int maxValue = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(
+                new ColorSupplier().getRandomColor(), new Random().nextInt(maxValue));
```

Comment:
```suggestion
public class Lottery {
    private static final int MAX_VALUE = 100;
    private static final ColorSupplier colorSupplier = new ColorSupplier();
    private static final Random random = new Random();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE));
```
Constant values should be private static final and in uppercase
Good practice to move objects like ColorSupplier, Random to class fields (later you are gonna learn about it more)